docs/docsite/rst/dev_guide/developing_collections_structure.rst

```diff
@@ -52,11 +52,28 @@ A collection must have a ``galaxy.yml`` file that contains the necessary informa
 .. _collections_doc_dir:
 
 docs directory
----------------
+--------------
+
+Use the ``docs`` folder to describe how to use the roles and plugins the collection provides, role requirements, and so on.
+
+For certified collections, Automation Hub displays documents written in markdown in the main ``docs`` directory.
```

Comment:
```suggestion
For certified collections, Automation Hub displays documents written in markdown in the main ``docs`` directory with no subdirectories. This will not display on https://docs.ansible.com.
```